l10n_be_codabox/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================================
+Codabox
+======================================
+
+This addons extends *Belgium - Import Bank CODA Statements* to import CODA from Codabox software.
+
+Installation
+============
+
+No specific installation step required
+
+Usage
+=====
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/119/10.0
+
+
+Known issues / Roadmap
+======================
+
+N/A
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/l10n-belgium/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
+l10n-belgium/issues/new?body=module:%20
+l10n_be_codabox%0Aversion:%20
+10.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Sam Lefever  <https://www.niboo.com>
+* Pierre Faniel <https://www.niboo.com>
+* Alexandre Dutry <https://www.niboo.com>
+* Antoine Verlant <https://www.niboo.com>
+
+Maintainer
+----------
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit https://odoo-community.org.
+
```

Comment:
Could you split this README file in fragments following to the [template](https://github.com/OCA/maintainer-tools/tree/master/template/module/readme).
The README.rst will the be generated by the OCA bot.